tests/Microsoft.Identity.Test.Unit/PublicApiTests/AuthenticationResultTests.cs

```diff
@@ -52,6 +55,111 @@ public void GetAuthorizationHeader()
             Assert.AreEqual("SomeTokenType at", ar.CreateAuthorizationHeader());
         }
 
+        [TestMethod]
+        public void GetIdTokenClaimsTest()
```

Comment:
Maybe add a check on some of the integration tests.

Reply:
Working on it